.eslintrc.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
 			'stylelint',
 			'@wordpress/stylelint-config',
 			'@pmmmwh/react-refresh-webpack-plugin',
+			'react-transition-group',
```

Comment:
❓ Just curiosity. Not an issue. Why is needed to add this dependency here?

Reply:
This PR uses the same `react-transition-group` dependency as the one `@woocommerce/components` depends on, so `react-transition-group` is not listed in this repo's package.json file. Therefore, this change is to let eslint know it's one of the core modules and to prevent from the eslint error `import/no-extraneous-dependencies`.